src/test/java/br/ufpe/cin/mergers/handlers/TypeAmbiguityErrorHandlerTest.java

```diff
@@ -60,9 +60,26 @@ public void testImportPackageMember() {
 				new File("testfiles/importpackagemember/right/Test/src/Test.java"),
 				null);
 		String mergeResult = FilesManager.getStringContentIntoSingleLineNoSpacing(ctx.semistructuredOutput);
-		
-		assertTrue(mergeResult.contains("<<<<<<<MINEimportpckt.A;=======importpcktright.*;>>>>>>>YOURS"));
-		assertTrue(ctx.typeAmbiguityErrorsConflicts==1);
+
+        assertThat(mergeResult).contains("<<<<<<<MINEimportpckt.A;=======importpcktright.*;>>>>>>>YOURS");
+        assertThat(ctx.typeAmbiguityErrorsConflicts).isOne();
+	}
+
+	@Test
+	public void testTypeAmbiguityErrorParameter() {
+		boolean defaultValue = JFSTMerge.isTypeAmbiguityHandlerEnabled;
+
+		JFSTMerge.isTypeAmbiguityHandlerEnabled = false;
+		MergeContext ctx = 	new JFSTMerge().mergeFiles(
+				new File("testfiles/importmembermember/left/Test/src/Test.java"),
+				new File("testfiles/importmembermember/base/Test/src/Test.java"),
+				new File("testfiles/importmembermember/right/Test/src/Test.java"),
+				null);
+		String mergeResult = FilesManager.getStringContentIntoSingleLineNoSpacing(ctx.semistructuredOutput);
+
+        assertThat(mergeResult).contains("importjava.util.List;importjava.awt.List;publicclassTest{Listlist;}");
+        assertThat(ctx.typeAmbiguityErrorsConflicts).isZero();
```

Comment:
Indentação :eyes:

Reply:
@gaabs and @jvcoutinho is there an identation verification or formatting tools that we could add to our travis script so that this could be automatically checked at integration time?

Reply:
@pauloborba Probably these indentation errors were caused by Git or my IDE during the push, I don't know. It's weird because I don't see any of these in my local files:

![2018-09-27 1](https://user-images.githubusercontent.com/27645414/46147380-c0e6a480-c23b-11e8-842f-24b31a4ba266.png)

Reply:
The problem was the IDE. The indentation was made with 4 spaces instead of tabs.